FirebaseMessaging/Sources/FIRMessagingConstants.h

```diff
@@ -31,9 +31,52 @@ FOUNDATION_EXPORT NSString *const kFIRMessagingMessageLinkKey;
 FOUNDATION_EXPORT NSString *const kFIRMessagingRemoteNotificationsProxyEnabledInfoPlistKey;
 FOUNDATION_EXPORT NSString *const kFIRMessagingSubDirectoryName;
 
-// Notifications
+#pragma mark - Notifications
+
 FOUNDATION_EXPORT NSString *const kFIRMessagingCheckinFetchedNotification;
 FOUNDATION_EXPORT NSString *const kFIRMessagingAPNSTokenNotification;
+FOUNDATION_EXPORT NSString *const kFIRMessagingDefaultGCMTokenFailNotification;
 FOUNDATION_EXPORT NSString *const kFIRMessagingRegistrationTokenRefreshNotification;
 
+FOUNDATION_EXPORT const int kFIRMessagingSendTtlDefault;  // 24 hours
+
+/**
+ *  Value included in a structured response or GCM message from IID, indicating
```

Comment:
The description doesn't make sense. I updated. But yeah FM code-wise should not depend on IID. But might still handle some signals sending from IID in case users still use IID.